packages/@aws-cdk/aws-cloudwatch/lib/graph.ts

```diff
@@ -641,7 +653,46 @@ export interface HorizontalAnnotation {
 }
 
 /**
- * Fill shading options that will be used with an annotation
+ * Vertical annotation to be added to a graph
+ */
+export interface VerticalAnnotation {
+  /**
+   * The date and time in the graph where the vertical annotation line is to appear
+   */
+  readonly value: string;
```

Comment:
If this is supposed to be a timestamp, I think we should type it that way instead of as a string.

Reply:
Also, should this be called `date` instead?

Reply:
done